sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -0,0 +1,935 @@
+use bloodhound::results;
```

Comment:
```suggestion
use bloodhound::results::{self, Checker, CheckStatus, CheckerMetadata};
```

That would cut out some boilerplate but still let you use `result::Mode` or whatever in places where the type might be ambiguous.